### PR TITLE
docs: add shared props

### PR DESCRIPTION
**Pull request by bear-junior**

### CODE REVIEW

**Review by maciejBudzinskiNG — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -62,37 +62,55 @@ export default TestScreen
...
-| `foregroundImage`             |`oneOfType([object, number])`                          |    Yes   |                                                                                                                                                                                                | Set image in the foreground                              |
-| `titleStyle`                  |`Text.propTypes.style`                                 |    Yes   |                                                                                                                                                                                                | Set style for text in foreground                         |
+| `onRef`                 |`func`                                                 |    Yes   |                                                                                                                                                                                                | Reference callback. You can call goToPage(pageNumber) method through ref to programatically navigate to given tab                |
```
Comment (davidwico) [inline README.md:102]: Typo. Should be `programmatically`, double `m`

File: README.md [hunk 2]
```diff
@@ -62,37 +62,55 @@ export default TestScreen
...
+| :---------------------------: | :---------------------------------------------------: | :-------:| :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------:|
+ | `foregroundImage`             |`oneOfType([object, number])`                          |    Yes   |                                                                                                                                                                                                | Set image in the foreground                              |
+| `header`                      |`func`                                                 |    Yes   |                                                                                                                                                                                                | Fuction that renders custom header                       |
```
Comment (davidwico) [inline README.md:97]: Typo. Should be `Function`, missing `n`

**Review by davidwico — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bear-junior) [reply README.md]: Thanks! :)

File: README.md (hunk 2 above)
Comment (bear-junior) [reply README.md]: Thanks :)

**Review by davidwico — APPROVED** [review]

**Review by pi0trpietruszewski — APPROVED** [review]

**Pull request merged**